src/components/enter-mfa/index.njk

```diff
@@ -11,8 +12,15 @@
 
   <h1 class="govuk-heading-l govuk-!-margin-top-0 govuk-!-margin-bottom-3">{{'pages.enterMfa.header' | translate }}</h1>
 
+  {% set insetText %}
+    {{ 'pages.checkYourPhone.info.paragraph1' | translate }} <span class='govuk-!-font-weight-bold'>{{ phoneNumber.slice(-3) }}</span>
```

Comment:
The resource is coming from pages.checkYourPhone but everything else on the page usese pages.enterMfa.  Is that right?